packages/datatrak-web/src/components/index.ts

```diff
@@ -5,5 +5,6 @@
 
 export { PageContainer } from './PageContainer';
 export { ButtonLink } from './ButtonLink';
+export { CodeGeneratorQuestion } from './CodeGeneratorQuestion/CodeGeneratorQuestion';
```

Comment:
Just noticed this - can we put this in the `Questions` folder with the other question types?